aldryn_newsblog/models.py

```diff
@@ -175,6 +176,6 @@ def __str__(self):
         return u'Latest entries: {0}'.format(self.latest_entries)
 
     def get_articles(self):
-        articles = Article.objects.active_translations(get_language()).filter(
-            app_config=self.app_config)
+        articles = Article.published.active_translations(
+            get_language()).filter(app_config=self.app_config)
```

Comment:
I think, it was working for me. Do you have a similar test case from another application?